docs/usage/configuration-options.md

```diff
@@ -777,6 +777,19 @@ Example:
 }
 ```
 
+```json
+{
+  "customManagers": [
+    {
+      "customType": "jsonata",
+      "matchStrings": [
+        "packages.{ \"depName\": package, \"currentValue\": version }"
+      ]
+    }
+  ]
+}
+```
```

Comment:
Isn't this - and the example above it which you copy/pasted from - an incomplete `customManagers` example? e.g. missing `fileMatch`